editor/src/clj/editor/model.clj

```diff
@@ -370,66 +413,69 @@
+                                                                                     :type resource/Resource
+                                                                                     :value (cond-> texture texture-binding-should-be-deleted (or fake-resource))
+                                                                                     :prop-kw :texture
+                                                                                     :error (when texture-binding-should-be-deleted
+                                                                                              (g/->error _node-id :texture :warning texture
+                                                                                                         (format "'%s' is not defined in the material. Clear the field to delete it."
+                                                                                                                 sampler)))
+                                                                                     :edit-type {:type resource/Resource
+                                                                                                 :ext (conj image/exts "cubemap")
+                                                                                                 :clear-fn (fn [_ _] (g/delete-node _node-id))}}
+                                                                                    texture-binding-should-be-deleted
+                                                                                    (assoc :original-value fake-resource))])
+                                                                         ;; texture binding does not exist
+                                                                         (let [sampler (key sampler-name+order)]
+                                                                           [texture-binding-prop-key
+                                                                            {:node-id material-binding-node-id
+                                                                             :label sampler
+                                                                             :value nil
+                                                                             :type resource/Resource
+                                                                             :edit-type {:type resource/Resource
+                                                                                         :ext (conj image/exts "cubemap")
+                                                                                         :set-fn (fn [_ _ _ new] (create-texture-binding-tx material-binding-node-id sampler new))}}])))))
+                                                                 (sort-by key all-sampler-name+orders))]
+                          (vec (concat combined-material-properties material-attribute-properties)))
```

Comment:
`(into combined-material-properties material-attribute-properties)` works, since `combined-material-properties` is already a vector we can append `material-attribute-properties` `into`.